bin/load-runner.js

```diff
@@ -98,6 +104,10 @@ var outputDir = null;
 var saveOutput = false;
 var writeInProgess = 0;
 
+//initialize random number generator
+const seed = parseInt(args.seed);
+const rand = isNaN(seed) ? lcg() : lcg(seed);
```

Comment:
Like I mentioned [in the other PR](https://github.com/feedhenry/load-runner/pull/3#discussion_r102909250), there's still the possibility of this blowing up if `seed < 1` (numbers that aren't positive are not `NaN`, but `lcg` will throw a `TypeError`).